hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -246,7 +254,12 @@ private void resolveMutableBuilderAttributes(
             builder.autoRenewAccountId(resolvedExpiryMeta.autoRenewAccountId());
         }
         if (op.hasFeeScheduleKey()) {
-            builder.feeScheduleKey(op.feeScheduleKey());
+            final var newSubmitKey = op.feeScheduleKey();
```

Comment:
Rename newSubmitKey to newFeeScheduleKey so we don't confuse with real submitKey.